tools/ci/tests/min_cam_rate_test.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
+
+import numpy as np
+import pandas as pd
+
+from tools.ci.common import TestOptions, Decorators
+from tools.ci.sim_results import SimRecordedData
+
+
+@Decorators.artery_test
+@Decorators.with_omnetpp_settings({
+    '**.transmission.result-recording-modes': 'all',
+    '**.transmission.result-recording-modes': 'all'
+})
+@Decorators.defines_test_options({
+    'min_cam_rate': 0.1,
+    'max_cam_rate': 1
+})
+def min_cam_rate_test(test: TestCase, data: SimRecordedData, test_options: TestOptions):
+    generated_deltas = data.vector[data.vector['vectorName'] == 'transmission:vector(camGenerationDeltaTime)']
```

Comment:
I think you can make your life a lot easier by just reading the recorded simulation time stamp for each transmission. `CaService`'s transmission statistic is recorded at the source, i.e. it is not affected by any delays at lower layers, which may violate the generation rate constraints.